SJH/Day2/가운데들자의합.js

```diff
@@ -0,0 +1,8 @@
+function solution(s) {
+    var answer = '';
+    if (s.length % 2 === 0)
+    {
+        return s.substr((s.length/2)-1,2)
+    } else
+        return s[Math.floor(s.length/2)]
```

Comment:
```suggestion
    if (s.length % 2 === 0) {
        return s.substr((s.length / 2) - 1, 2)
    } else {
        return s[Math.floor(s.length / 2)]
    }
```

제어문을 사용할 때에는 중괄호를 붙여 코딩 컨벤션을 준수해주세요! 그럼에도 else 밑으로 작성하신 코드가 돌아가는 이유는 if-else 제어문에서 문장을 한줄만 적었을 경우에도 실행되기 때문입니다. (2줄 이상 불가능)